### PR TITLE
Call ride

### PR DESCRIPTION
**Pull request by zdcthomas**
____ Wrote Tests
__everyone__ Implemented
____ Reviewed
# Neccesary checkmarks:
- [] All Tests are Passing

- [] The code will run locally

## Type of change
- [] New feature
- [] Bug Fix

# Implements/Fixes:
* description
closes #

# Check the correct boxes

- [] This broke nothing
- [] This broke some stuff
- [] This broke everything

# Testing Changes
- [] No Tests have been changed
- [] Some Tests have been changed
- [] All of the Tests have been changed(Please describe what in the world happened)

# Checklist:

- [] My code has no unused/commented out code
- [] I have reviewed my code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have fully tested my code

(╯°□°）╯︵ ┻━┻

# Please Include a link to a gif of your feelings about this branch
Link:

### CODE REVIEW

File: src/components/Ride.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+  async handleYesClick() {
+    let origin = this.props.origin
+    let destination = {
+      latitude: this.props.adventure.destination.latitude,
+      longitude: this.props.adventure.destination.longitude
+    }
+    let cost_token = this.props.adventure.price_range.cost_token
+    let ride = await ApiService.createRide({ origin: origin, destination: destination, cost_token: cost_token })
+    // ggo off and show them the shit
```
Comment (jude-lawson) [inline src/components/Ride.js]: Can we remove these comments?

File: src/components/LoginPage.js [hunk 2]
```diff
@@ -0,0 +1,202 @@
+import React, { Component } from 'react';
```
Comment (jude-lawson) [inline src/components/LoginPage.js]: Can we remove this file?

File: src/components/LandingPage.js [hunk 3]
```diff
@@ -0,0 +1,178 @@
+import React, { Component } from 'react';
```
Comment (jude-lawson) [inline src/components/LandingPage.js]: Can we remove this file?

File: src/components/EstimatePage.js [hunk 4]
```diff
@@ -0,0 +1,134 @@
+import React, { Component } from 'react';
```
Comment (jude-lawson) [inline src/components/EstimatePage.js]: Can we remove this file?

**Review by jude-lawson — CHANGES_REQUESTED** [review]

**Review by hmesander — COMMENTED** [review]
Looks good.

**Pull request merged**